### ISSUE TITLE
suggestion: only show the mascotte profile on mobile /signup

### ISSUE THREAD

**Issue by smeijer** [kentcdodds/remix-kentcdodds#86]
The huge cards are nice, but take a lot of space. It's worth exploring to only show its face when rendered on mobile. If those 3 faces would fit next to each other, they'd require a lot less scrolling.

**Comment by kentcdodds** [comment]
Oh, good idea 👍

**Comment by kentcdodds** [comment]
Yeah, still want to do this as well. We'll get the new Kody mascot soon (hopefully) and then can work on this.
